src/Illuminate/Support/helpers.php

```diff
@@ -523,3 +523,16 @@ function with($value, ?callable $callback = null)
         return is_null($callback) ? $value : $callback($value);
     }
 }
+
+if (! function_exists('to_english_numbers')) {
+    /**
+     * Converting Persian/Arabic digits to English
+     * @param  mixed  $value
```

Comment:
Is this really `mixed`? Shouldn't this be

```suggestion
     * @param  string  $value
```
?

Reply:
I created this function to Number helper so we don't need to repeat it here. I'll remove that from helper.php